src/Sulu/Bundle/MediaBundle/Resources/js/containers/Form/fields/MediaSelection.js

```diff
@@ -16,12 +18,9 @@ export default class MediaSelection extends React.Component<FieldTypeProps<Value
 
     render() {
         const {formInspector, disabled, value} = this.props;
-
-        if (!formInspector || !formInspector.locale) {
-            throw new Error('The media selection needs a locale to work properly');
-        }
-
-        const {locale} = formInspector;
+        const locale = formInspector && formInspector.locale
```

Comment:
Is the check for `formInspector` really necessary? I think the `formInspector` should always be there, so only `formInspector.locale` should be enough...